web/main/models.py

```diff
@@ -2431,18 +2401,19 @@ def related_resources(self):
         return Resource.objects.filter(resource_id=self.id, resource_type='TextBlock')
 
 
-class User(NullableTimestampedModel, AbstractBaseUser):
+class User(NullableTimestampedModel, PermissionsMixin, AbstractBaseUser):
     email_address = models.CharField(max_length=255, unique=True)
     attribution = models.CharField(max_length=255, default='Anonymous', verbose_name='Display name')
     affiliation = models.CharField(max_length=255, blank=True, null=True)
-    verified_email = models.BooleanField(default=False)
     verified_professor = models.BooleanField(default=False)
     professor_verification_requested = models.BooleanField(default=False)
 
-    # used to assign super_admin or case_admin status
-    roles = models.ManyToManyField(Role,
-        through=RolesUser
-    )
+    is_staff = models.BooleanField(default=False)
+    is_active = models.BooleanField(default=False)
+
+    # differentiate between real User model and AnonymousUser model:
+    is_authenticated = True
+    is_anonymous = False
```

Comment:
@jcushman can you say a word about why we have to set `is_authenticated` and `is_anonymous` manually? I should be able to figure out, but I'm not getting it.

Reply:
Looks like we don't need to anymore! When those two lines were added we weren't inheriting from `AbstractBaseUser` yet.

Reply:
Aha! Thanks!! I'll delete these lines too!

Reply:
Deleted. And, corrected the reverse migration per discussion in Slack.